include/pisa/wand_data.hpp

```diff
@@ -48,21 +50,37 @@ class wand_data {
 
         typename block_wand_type::builder builder(coll, params);
 
-        for (auto const &seq : coll) {
-            size_t term_occurrence_count = std::accumulate(seq.freqs.begin(), seq.freqs.end(), 0);
-            term_occurrence_counts.push_back(term_occurrence_count);
-            term_posting_counts.push_back(seq.docs.size());
+        {
+           pisa::progress progress("Storing terms statistics", coll.size());
+           size_t term_id = 0;
+           for (auto const &seq : coll) {
+               if(dropped_term_ids.find(term_id) != dropped_term_ids.end()){
```

Comment:
Side note: did you know that c++20 finally added `contains` method to sets? So progressive.